test/date-time/date-time-create.test.ts

```diff
@@ -0,0 +1,1068 @@
+import * as Assert from "assert";
+import { DateTime } from "../../src/date-time";
+import { DateTime as LuxonDateTime } from "luxon";
+
+
+suite("DateTime Create", () =>
+{
+    suite("Constructor", () =>
+    {
+        const validValue = "2024-01-01 10:00";
+
+        function checkIsInvalid(value: string, zone = "utc"): void
+        {
+            let dateTime = "";
+            try
+            {
+                dateTime = new DateTime({ value, zone }).toString();
+            }
+            catch (e: any)
+            {
+                // console.log(e.reason);
+                Assert.ok(e.reason);
+                return;
+            }
+
+            Assert.fail(`DateTime ${dateTime} is valid`);
+        }
+
+        test(`Given a valid value and zone
+        when the constructor is called 
+        then it should return a DateTime object`,
+            () =>
+            {
+                Assert.ok(new DateTime({ value: validValue, zone: "utc" }));
+            }
+        );
+
+        test(`Given value as empty string and a valid zone
+        when the constructor is called 
+        then it should throw a validation error`,
+            () =>
+            {
+                checkIsInvalid("");
```

Comment:
You should do this test with Assert.throws, there you can also check the error thrown is the error exepcted